testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinMultiplatformPluginSmokeTest.groovy

```diff
@@ -28,7 +29,7 @@ class KotlinMultiplatformPluginSmokeTest extends AbstractKotlinPluginSmokeTest {
     def 'test kotlin multiplatform with js project (kotlin=#kotlinVersion)'() {
         given:
         withKotlinBuildFile()
-        useSample("kotlin-multiplatform-js-example")
+        useSample("kotlin-multiplatform-js-jvm-example")
```

Comment:
Did you rename the sample because it is a kmp project with js and jvm targets?

Reply:
That happend unintentional, I'll revert it.